src/components/navbar.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+
+const Navbar = () => {
+  const links = [
+    {
+      id: 1,
+      path: '/',
+      text: 'Rockets',
+    },
+    {
+      id: 2,
+      path: '/Missions',
+      text: 'Missions',
+    },
+    {
+      id: 3,
+      path: 'MyProfile',
+      text: 'MyProfile',
+    },
+  ];
+
+  return (
+    <nav className="Nav-bar">
```

Comment:
**[OPTIONAL]:** 

- [ ] For unique styling like a navbar on a page, you can consider using `id` instead of className :+1: